.gitlab-ci.yml

```diff
@@ -26,6 +26,13 @@ maxset:
     paths:
     - build/
 
+clang-static:
+  stage: build
+  image: espressomd/espresso-ubuntu-clang-cuda:latest
+  script:
+    - export myconfig=maxset with_coverage=false with_static_analysis=true
+    - bash maintainer/cuda_build.sh
```

Comment:
We need to save the log as an artefact.  Otherwise we cannot access the result of the static analysis which makes it a bit pointless.

Reply:
Actually we can’t do that. If errors are found, the build fails, meaning we can’t upload artifacts. If there are no errors, there is nothing to upload.